create/mysql_test.go

```diff
@@ -27,8 +27,8 @@ func TestMySQL(t *testing.T) {
 	}{
 		{
 			name:   "simple",
-			create: mySQLCmd{},
-			want:   storage.MySQLParameters{},
+			create: mySQLCmd{SSHKeys: []storage.SSHKey{""}, AllowedCidrs: []storage.IPv4CIDR{}},
```

Comment:
I refactored the test a bit to overcome the repetitions of slice initialisations